.github/workflows/pypi-publish.yml

```diff
@@ -2,7 +2,6 @@ name: Upload Python Package
 
 on:
   workflow_dispatch:
-    branches: [main]
```

Comment:
Ooh, do you not need to specify a branch for this any more? (And if so, should you remove the `:` after `workflow_dispatch`?)

Reply:
I don't think it was ever valid syntax, my IDE yelled at me. I think the `:` is still needed though?

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_dispatch

Reply:
🤷‍♂️